gssapi/_win_config.py

```diff
@@ -41,14 +42,31 @@ def error_not_found():
 def configure_windows():
     """
     Validate that KfW appears to be installed correctly and add it to the
-    PATH if necessary. In the case that it can't be located, raise an error.
+    DLL directories/PATH if necessary. In the case that it can't be located,
+    raise an error.
     """
-    if k4w_in_path():
+    if kfw_available():
         return  # All set, necessary DLLs should be available
+
     if os.path.exists(KFW_BIN):  # In standard location
-        os.environ['PATH'] += os.pathsep + KFW_BIN
-        if k4w_in_path():
+        try:  # to use Python 3.8's DLL handling
+            os.add_dll_directory(KFW_BIN)
+        except AttributeError:  # <3.8, use PATH
+            os.environ['PATH'] += os.pathsep + KFW_BIN
+        if kfw_available():
             return
+
+    # Check if kinit is in the PATH which should lead us to the bin folder
+    kinit_path = shutil.which('kinit')  # MIT exclusive binary
+    if kinit_path:  # Non-standard install location
+        try:  # Most likely >=3.8, otherwise it would have been found already
+            os.add_dll_directory(os.path.dirname(kinit_path))
+        except AttributeError:  # <3.8, corrupted installation?
+            pass
+        else:
+            if kfw_available():
```

Comment:
Collapse this into `elif`.

Reply:
That's a try block, not an if statement:P At first I was like, try blocks support elif??? but they don't:(

Reply:
Welp, TIL.  Weird inconsistency :/